.travis.yml

```diff
@@ -33,6 +33,7 @@ script:
   - ws exec composer test-quality
   - ws exec composer test-unit
   - ws exec composer test-acceptance
+  - find . -type f ! -path "./.git/*" ! -path "./bin/workspace" ! -path "./bin/build" ! -path "./vendor/*" ! -name "*.phar" ! -name "*.php" -and \( -name "*.sh" -or -perm -0111 \) -exec shellcheck --exclude=SC1008,SC1091 {} +
```

Comment:
I think the paths are incorrect, it should be something like `find . -type f ! -path "./.git/*" ! -path "./dist/*" ! -path "./test/*" ! -name "*.xml" ! -name "*.txt" ! -name "*.properties" -and \( -name "*.sh" -or -perm -0111 \) -exec shellcheck --exclude=SC1008,SC1091 {} +`

Reply:
I'd argue the solr config should not be executable, but we can deal with that in another PR.